revolv/payments/tests/models/test_payment_instrument_type.py

```diff
@@ -1,12 +1,16 @@
 from django.test import TestCase
-from revolv.payments.models import INSTRUMENT_PAYPAL, PaymentInstrumentType
+from revolv.payments.models import (INSTRUMENT_CHECK, INSTRUMENT_PAYPAL,
+                                    INSTRUMENT_REINVESTMENT,
+                                    INSTRUMENT_REPAYMENT,
+                                    PaymentInstrumentType)
 
 
 class PaymentInstrumentTypeTest(TestCase):
 
-    fixtures = ['supported_instruments.json']
-
     def test_get_payment_instrument_types(self):
 
-        self.assertEqual(1, len(PaymentInstrumentType.objects.all()))
+        self.assertEqual(4, len(PaymentInstrumentType.objects.all()))
         self.assertEqual(INSTRUMENT_PAYPAL, PaymentInstrumentType.objects.first().name)
+        self.assertEqual(INSTRUMENT_REINVESTMENT, PaymentInstrumentType.objects.get(id=2).name)
```

Comment:
If you do leave the IDs in, it might be worth considering making small functions/constants to abstract out the ID so the test can read something like

    self.assertEqual(INSTRUMENT_REINVESTMENT, reinvestment_payment().name)

Which is a little clearer to grok on a quick pass through the test.